taskcluster/ci/generate-screenshots/kind.yml

```diff
@@ -0,0 +1,50 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - ffios_taskgraph.transforms.screenshots:transforms
+    - ffios_taskgraph.transforms.secrets:transforms
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+
+job-defaults:
+    description: Generate localized screenshots by delegating the work to bitrise.io
+    run:
+        using: run-commands
+        use-caches: false
+        secrets:
+            by-level:
+                '3':
+                    - name: project/mobile/firefox-ios/bitrise
```

Comment:
I created this secret and put my personal token in it. This is the process I used https://devcenter.bitrise.io/getting-started/account-security/#generating-personal-access-tokens-manually